elements/pfe-jump-links/src/pfe-jump-links-nav.html

```diff
@@ -0,0 +1,47 @@
+<!-- <nav>
+  <h2 class="sr-only" hidden>Page navigation</h2>
+  ${ this.has_slot("heading") ?
+  `<slot class="heading" name="heading" id="heading"></slot>`
+  : `<h4 class="heading" id="heading">Jump to section</h4>`
+  }
+  ${ this.has_slot("logo") ?
+  `<slot class="logo" name="logo" id="logo"></slot>`
+  : ``
+  }
+
+  <div id="container"></div>
+  ${ this.has_slot("link") ?
+  `<slot class="link" name="link" id="link"></slot>`
+  : ``
+  }
+</nav> -->
+<!-- ^^ old version -->
+
+${this.hasAttribute("horizontal") ? `` :
+`<pfe-accordion>
+  <pfe-accordion-header>
+  </pfe-accordion-header>
+  <pfe-accordion-panel>`
+}
+    <nav>
+      <h2 class="sr-only" hidden>Page navigation</h2>
```

Comment:
How will the embedded content be translated?

Reply:
That's a really good question. What approach are we taking in other components? That all text should come from the light DOM?

This is an accessibility-driven enhancement that I think a lot of devs would leave out on accident, so I didn't want it to have to be in the light DOM, but if need be then so be it.